app/Http/Controllers/Api/WaitingRoomsController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use Gladiator\Models\WaitingRoom;
+use Illuminate\Http\Request;
+
+class WaitingRoomsController extends ApiController
+{
+    public function calculateSplit(Request $request)
```

Comment:
Didn't realize this was a big deal as it only returns user ID's. I can look into adding the CSRF middleware